lupus_to_mqtt/sensor/Shutter.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from lupus_to_mqtt.MQTT import MQTT
+from lupus_to_mqtt.Logger import Logger
+from lupus_to_mqtt.Connection import Connection
+from . import Sensor
+
+
+class Shutter(Sensor):
+    """Class representing a shutter."""
+
+    def __init__(self, data, panel):
+        self._onOff = int(data.get('onOff'))
+        self._level = int(data.get('level'))
+        super().__init__(data, panel)
+
+        self._logger = Logger.getInstance()
+        self._connection = Connection.getInstance()
+
+    def registerDevice(self):
+        mqtt = MQTT.getInstance()
+        msg_config = json.dumps({
+            "name": self.name,
+            "unique_id": self._id,
+            "availability_topic": self._panel.device_name + "/availability",
+            "state_topic": f'{self._panel.device_name}/{self._id}/state',
+            "command_topic": f'{self._panel.device_name}/{self._id}/set',
+            "device": {
+                "identifiers": self._sid,
+                "manufacturer": self._panel.manufacturer,
+                "name": self.name,
+            }
+        }
+        )
+        mqtt.publish_message(f'homeassistant/switch/{self._id}/config', msg_config)
```

Comment:
I think this should be an MQTT "cover" instead of a switch, as it can have multiple positions (If I understand this correctly; unfortunately, I don't have a shutter relay to test it myself).

Reply:
See: https://www.home-assistant.io/integrations/cover.mqtt/